README.md

```diff
@@ -1,23 +1,34 @@
+[<img src="docs/logo.png" alt="Lagom" width="50%">](https://github.com/lagom/lagom)
+
+[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/lagom/lagom?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [<img src="https://img.shields.io/travis/lagom/lagom.svg"/>](https://travis-ci.org/lagom/lagom)
+
 # INTRO
 
-This is a sample app of the Lagom Framework.
+Lagom is a Swedish word meaning just right, sufficient. Microservices are about creating services that are just the right size, that is, they have just the right level of functionality and isolation to be able to adequately implement a scalable and resilient system.
+
```

Comment:
Looks like this is missing a paragraph that is in the Scala version: https://github.com/lagom/online-auction-scala/pull/25/files#diff-04c6e90faac2675aa89e2176d2eec7d8R8